packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -358,3 +358,117 @@ export function updateOutputPort(node: WorkflowNode<any>, updatedOutputPort: Wor
 	if (!outputPort) return;
 	outputPort = Object.assign(outputPort, updatedOutputPort);
 }
+
+// Check if the current-state matches that of the output-state.
+// Note operatorState subsumes the keys of the outputState
+export const isOperatorStateInSync = (
```

Comment:
If this function doesn't use any out of scope information, why not making it a function?
```suggestion
export function isOperatorStateInSync(
```

Reply:
No particular reason other than most of stuff in the file are of the form `export const x = (...) => { ... }` to keep things consistent

Reply:
I see, am used to use `const` when I need access to reactive values from out of scope of a function.
But it does not matter.